data/json/recipes/recipe_others.json

```diff
@@ -4240,6 +4240,31 @@
       [ [ "spike", 2 ] ]
     ]
   },
+  {
+    "type": "recipe",
+    "result": "metal_smoking_rack",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_TOOLS",
+    "skill_used": "fabrication",
+    "difficulty": 4,
+    "time": 8000,
```

Comment:
It's just 8 minutes, I think that's way too fast.

Reply:
The time was based off the construction speed of the metal butchering rack. I can up the time needed for construction as appropriate. I just figured the recipe for the butchering rack would make a decent base.
![image](https://user-images.githubusercontent.com/49844315/56710930-7a382a80-66f6-11e9-85fb-8bf95343446f.png)

Reply:
Then butchering rack construction time should be increased too. I'd increase it to at least 20-30 minutes.

Reply:
Alright, I'll go ahead and change both to 30 min.

Reply:
Regarding the conversation later on in the pull request, if the requirements for the metal smoking rack creation and butchering rack creation includes welding tools I'm not opposed to changing both of these recipes to be huge time sinks. This would work much the same way as crafting chains where it could take a long amount of time (7+ hours) to create the deployable furniture with the tradeoff that it saves time in the future with its ability to be quickly (a)ctivated or disassembled.